lms/static/scripts/frontend_apps/components/FilePickerApp.tsx

```diff
@@ -169,16 +173,16 @@ export default function FilePickerApp({ onSubmit }: FilePickerAppProps) {
   return (
     <main>
       <form
-        className="FilePickerApp__form"
+        className="grid grid-cols-[auto_1fr] gap-4 w-[500px] max-w-[80vw] mx-auto mt-4"
```

Comment:
Don't worry, when we refactor this layout, we won't style the `form` element anymore :).